src/tagging-compliance/resource_parsing.py

```diff
@@ -111,4 +111,5 @@ def lambda_handler(event, context):
     push_to_gateway(
         os.environ["prometheus_ip"], job="TaggingResourceList", registry=registry
     )
+
```

Comment:
I can't specifically comment on line # 108 but here we have to update region information that is missing in your PR   resource_list_guage.labels(resource, lambda_item["Region"], account_id).set(
                0
            )

Reply:
updated https://github.com/XgridInc/xc3/commit/131ca5a9d982dca11cc2842e2da5fe3dcbf159eb